src/main/java/org/sonar/plugins/csharp/RoslynProfileExporter.java

```diff
@@ -138,10 +138,14 @@ public void exportProfile(RulesProfile rulesProfile, Writer writer) {
     appendLine(writer, "</RoslynExportProfile>");
   }
 
-  private static ImmutableMultimap<String, ActiveRule> activeRoslynRulesByPluginKey(List<ActiveRule> activeRules) {
+  public static ImmutableMultimap<String, ActiveRule> activeRoslynRulesByPluginKey(List<ActiveRule> activeRules) {
     ImmutableMultimap.Builder<String, ActiveRule> builder = ImmutableMultimap.builder();
 
     for (ActiveRule activeRule : activeRules) {
+      if (activeRule.getRule().getLanguage() != CSharpPlugin.LANGUAGE_KEY) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change this comparison to use the equals method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1698)